libs/feature/search/src/lib/utils/service/fields.spec.ts

```diff
@@ -480,17 +503,22 @@ describe('search fields implementations', () => {
       describe('#getAvailableValues', () => {
         let values
         beforeEach(async () => {
-          values = await lastValueFrom(searchField.getAvailableValues())
+          values = await lastValueFrom(
+            searchField.getAvailableValues(configFilters)
+          )
         })
         it('calls search with a simple unsorted terms', () => {
-          expect(repository.aggregate).toHaveBeenCalledWith({
-            'tag.default': {
-              type: 'terms',
-              limit: 1000,
-              field: 'tag.default',
-              sort: ['desc', 'count'],
+          expect(repository.aggregate).toHaveBeenCalledWith(
```

Comment:
weird formatting changes, I wonder how the `npm run format` does not say anything